pylearn2/costs/tests/test_term_1_0_l1_penalty.py

```diff
@@ -59,7 +59,8 @@ def test_correctness():
                                  N=2)])
 
     cost = SumOfCosts([(0.99, Default()),
-                       (0.01, L1WeightDecay((0., 1.0)))])
+                       (0.01, L1WeightDecay({'hidden': 0.,
+                                             'output': 1.0}))])
```

Comment:
Oh, but if you actually pass the parameters in a dictionary, it will not test the first bug anymore. You would have to pass an empty dictionary for that.